runtime/lib/node.h

```diff
@@ -86,7 +86,7 @@ typedef struct minicute_node_NStructureFields
 {
   int8_t tag;
   int32_t size;
-  int8_t *values;
+  int8_t *values[0];
```

Comment:
It uses "Struct Hack". I'm not sure whether or not it should be replaced by `values[1]`, or even a pointer...
Let's use it as a part of temporary design choices.